core/src/consensus.rs

```diff
@@ -48,6 +51,8 @@ use {
     thiserror::Error,
 };
 
+pub const SUPERMINORITY_THRESHOLD: f64 = 1f64 / 3f64;
```

Comment:
If we're moving this here, it would be nice to consolidate all our hardcoded consensus constants in this file:
```
DUPLICATE_LIVENESS_THRESHOLD
DUPLICATE_THRESHOLD
VOTE_THRESHOLD_SIZE (terrible name btw)
SWITCH_FORK_THRESHOLD
VOTE_THRESHOLD_DEPTH
WAIT_FOR_SUPERMAJORITY_THRESHOLD_PERCENT
```